dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -1,41 +1,55 @@
 -# English only for now
 - if language == "en"
```

Comment:
@laurelfan I have been working to move this logic into the controller but I'm not sure how to call the controller method from the view, because I don't think that view is rendered with the SurveyResultController as its controller.  Thoughts?